config/txloader/load/harvestcraft/lang/pl_PL.lang

```diff
@@ -0,0 +1,1136 @@
+item.leekbaconsoupItem.name=Zupa z Pora i Bekonu
+item.herbbutterparsnipsItem.name=Pasternak w Maśle Ziołowym
+item.scallionbakedpotatoItem.name=Ziemniak Pieczony z Cebulką
+item.soymilkItem.name=Mleko Sojowe
+item.firmtofuItem.name=Twarde Tofu
+item.silkentofuItem.name=Jedwabiste Tofu
+
+item.bamboosteamedriceItem.name=Ryż z Bambusem na Parze
+item.roastedchestnutItem.name=Pieczony Kasztan
+item.sweetpotatosouffleItem.name=Suflet z Batatów
+item.cashewchickenItem.name=Kurczak z Nerkowcem
+item.apricotjuiceItem.name=Sok Morelowy
+item.apricotyogurtItem.name=Morelowy Jogurt
+item.apricotglazedporkItem.name=Glazurowana Wieprzowina z Morelą
+item.apricotjellyItem.name=Morelowy Dżem
+item.apricotjellysandwichItem.name=Kanapka z Dżemem Morelowym
+item.apricotsmoothieItem.name=Morelowe Smoothie
+item.figbarItem.name=Ciastko Figowe
+item.figjellyItem.name=Dżem Figowy
+item.figjellysandwichItem.name=Kanapka z Dżemem Figowym
+item.figsmoothieItem.name=Figowe Smoothie
+item.figyogurtItem.name=Figowy Jogurt
+item.figjuiceItem.name=Figowy Sok
+item.grapefruitjuiceItem.name=Sok z Grejpfruta
+item.grapefruitjellyItem.name=Dżem z Grejpfruta
+item.grapefruitjellysandwichItem.name=Kanapka z Dżemem z Grejpfruta
+item.grapefruitsmoothieItem.name=Smoothie z Grejpfruta
+item.grapefruityogurtItem.name=Jogurt z Grejpfruta
+item.grapefruitsodaItem.name=Napój z Grejpfruta
+item.citrussaladItem.name=Sałatka Cytrusowa
+item.pecanpieItem.name=Ciasto z Pekanem
+item.pralinesItem.name=Pralinki
+item.persimmonjuiceItem.name=Sok z Persymony
+item.persimmonyogurtItem.name=Jogurt z Persymony
+item.persimmonsmoothieItem.name=Smoothie z Persymony
+item.persimmonjellyItem.name=Dżem z Persymony
+item.persimmonjellysandwichItem.name=Kanapka z Dżemem z Persymony
+item.pistachiobakedsalmonItem.name=Pieczony Łosoś z Pistacjami
+item.baconwrappeddatesItem.name=Daktyle w Bekonie
+item.datenutbreadItem.name=Chleb Daktylowy
+item.maplesyruppancakesItem.name=Naleśniki z Syropem Klonowym
+item.maplesyrupwafflesItem.name=Gofry z Syropem Klonowym
+item.maplesausageItem.name=Kiełbaska z Syropem Klonowym
+item.mapleoatmealItem.name=Owsianka z Syropem Klonowym
+item.peachesandcreamoatmealItem.name=Owsianka z Brozkiwniami i Śmietaną
+item.cinnamonappleoatmealItem.name=Owsianka z Cynamonem i Jabłkami
+item.maplecandiedbaconItem.name=Kandyzowany Bekon w Syropie Klonowym
+item.toastsandwichItem.name=Kanapka Tostowa
+item.potatoandcheesepirogiItem.name=Pierogi Ruskie
+item.zeppoleItem.name=Pączki Włoskie
+item.sausageinbreadItem.name=Kiełbaska w Chlebie
+item.chocolatecaramelfudgeItem.name=Cukierki Czekoladowo Karmelowe
+item.lavendershortbreadItem.name=Lawendowe Herbatniki
+item.beefwellingtonItem.name=Polędwica Wellington
+item.epicbaconItem.name=Epicki Bekon
+item.manjuuItem.name=Manjuu
+
+item.chickengumboItem.name=Gumbo z Kurczakiem
+item.generaltsochickenItem.name=Kurczak Generała Tso
+item.californiarollItem.name=Sushi Kalifornijskie
+item.futomakiItem.name=Futo Maki
```

Comment:
tego się raczej nie tłumaczy

Reply:
O którym mowa? California Rolls? Mogę w sumie się zgodzić, zmienię

Reply:
futomaki to jeden z generalnych typów sushi, futomaki to "gruba rolka" zawierająca min 4 składniki, z kolei hosomaki ma max 2 składniki.

California roll to podtyp hosomaki (no a przynajmniej tak mi się wydaje)